microservice-consul-demo/microservice-consul-demo-catalog/src/main/resources/templates/layout.html

```diff
@@ -1,24 +1,27 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org"
-	xmlns:layout="http://www.ultraq.net.nz/web/thymeleaf/layout">
+<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/web/thymeleaf/layout">
 <head>
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 <title>Layout</title>
 <link rel="stylesheet"
 	href="./webjars/bootstrap/3.3.6/css/bootstrap.min.css" />
 <link rel="stylesheet"
 	href="./webjars/bootstrap/3.3.6/css/bootstrap-theme.min.css" />
+<style type="text/css">
+    .navbar ul { list-style: none; display: inline-block; padding: 0;}
+    .navbar ul li { display: inline-block; padding-right: .5em; }
+</style>
 </head>
 
 <body>
 	<div class="container">
 		<div class="navbar">
-			<div class="navbar-inner">
-				<a class="brand" href="/"> Home</a> <a class="brand"
-					href="https://github.com/ultraq/thymeleaf-layout-dialect">
-					Thymeleaf - Layout </a>
-
-			</div>
+			<ul class="navbar-inner">
+				<li><a class="brand" href="/">Home</a></li>
+				<li><a class="brand" href="list.html">List</a></li>
+				<li><a class="brand" href="searchForm.html">Search</a></li>
+				<li><a class="brand" href="https://github.com/ultraq/thymeleaf-layout-dialect">Thymeleaf - Layout</a></li>
```

Comment:
That's a question for @ewolff. As this is a public sample, he might have had a reason adding it in the first place